lib/sequent/support/database.rb

```diff
@@ -46,7 +46,7 @@ def self.execute_sql(sql)
 
       def self.create_schema(schema)
         sql = "CREATE SCHEMA IF NOT EXISTS #{schema}"
-        if user = ActiveRecord::Base.connection_config[:username]
+        if user = ActiveRecord::Base.connection_db_config.configuration_hash[:username]
```

Comment:
Is this backward compatible to earlier versions of AR?

Reply:
Good catch, added back support for the legacy config